ci/integrationtests/task.yaml

```diff
@@ -71,10 +71,9 @@ spec:
 
         echo "> Installing dependencies"
         apk update && apk add make
-        pip install pipenv
+        pip install scp
 
         cd $(resources.inputs.repo.path)/tests
-        pipenv install -dev
```

Comment:
this command would update dependencies, better use pipenv sync, see https://pypi.org/project/pipenv/

Reply:
hmm, you are referring to a deleted line?

Reply:
yep, still this was wrong

Reply:
It's only a copy of what you described in the resp. readme. Then I guess you'd might want to change it here: https://github.com/gardenlinux/gardenlinux/blame/main/tests/README.md#L15

Reply:
mea culpa, I'll fix the readme